### PR TITLE
[CS] Add `Installation`

### PR DESCRIPTION
**Pull request by Ondra-D**
Translated the installation page to czech.

### CODE REVIEW

File: wiki/Installation/cs.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Instalace
+
+*Viz také: [Instalace v systému macOS](/wiki/Installation/macOS)*
+
+Tato stránka vám řekne, jak zajistit, aby hra fungovala na vašem zařízení Windows. [osu!academy](/wiki/osu!academy) má [video tutorial](https://www.youtube.com/watch?v=0V5GwzmMhpU), který vysvětlí, jak nainstalovat osu! na Windows.
+
+## Minimální systémové požadavky
+
+- .NET framework 4+ (instalační program by vás měl vyzvat ke stažení, v opačném případě jej najdete [zde](https://www.microsoft.com/en-us/download/details.aspx?id=48130))
```
Comment (TPGPL) [inline wiki/Installation/cs.md:9]:
```suggestion
- .NET framework 4+ (instalační program by vás měl vyzvat ke stažení, v opačném případě jej najdete [zde](https://www.microsoft.com/cs-CZ/download/details.aspx?id=48130))
```

File: wiki/Installation/cs.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Přejděte na [stránku pro stažení](https://osu.ppy.sh/home/download) a stáhněte si instalační program.
+2. Vyhledejte a spusťte instalační program.
+3. Zatímco je instalační program spuštěn, existuje desetisekundová perioda, která vám umožní vybrat umístění instalace.
+   - Pokud to chcete změnit, klikněte na text pod slovem „Ahoj!“ A vyberte umístění instalace.
+   - osu! bude nainstalován na toto místo, až bude časovač dokončen.
+4. Když je instalační program dokončen, osu! se zapne. osu! direct bude použit ke stažení některých startovacích beatmap pro vás.
+5. osu! vás vyzve k přihlášení nebo registraci.
+   - Pokud již účet máte, přihlaste se.
+   - Pokud se potřebujete zaregistrovat, podívejte se na [Registrace](/wiki/Registrace).
```
Comment (TPGPL) [inline wiki/Installation/cs.md:22]:
```suggestion
   - Pokud se potřebujete zaregistrovat, podívejte se na [Registrace](/wiki/Registration).
```

File: wiki/Installation/cs.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+**Pokud instalační program hry při pokusu o instalaci hry vyvolá chyby, můžete vyzkoušet několik věcí**
+
+- Zkuste přeinstalovat nejnovější verzi .NET Framework. [Najdete ho zde](https://dotnet.microsoft.com/download).
+- Vyčistěte počítač - skenujte pomocí antiviru, vyčistěte registry. Dobrý program pro obecné čištění je CCleaner. Naskenujte soubory v počítači i registry. Pokud existují položky registru z předchozích kopií osu, instalační program osu! může selhat ve vašem systému.
+
+## Přidávání beatmap
+
+*Viz také: [Beatmapy](/wiki/Beatmap)*
+
+Existují dva způsoby, jak přidat beatmapy, přes webovou stránku nebo přes osu! Direct (vyžaduje [osu! Supporter](/wiki/osu!Supporter)).
```
Comment (TPGPL) [inline wiki/Installation/cs.md:39]:
```suggestion
Existují dva způsoby, jak přidat beatmapy, přes webovou stránku nebo přes osu! Direct (vyžaduje [osu! Supporter](/wiki/osu!supporter)).
```

File: wiki/Installation/cs.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+1. Přejděte na [stránku pro stažení](https://osu.ppy.sh/home/download) a stáhněte si instalační program.
+2. Vyhledejte a spusťte instalační program.
+3. Zatímco je instalační program spuštěn, existuje desetisekundová perioda, která vám umožní vybrat umístění instalace.
+   - Pokud to chcete změnit, klikněte na text pod slovem „Ahoj!“ A vyberte umístění instalace.
+   - osu! bude nainstalován na toto místo, až bude časovač dokončen.
+4. Když je instalační program dokončen, osu! se zapne. osu! direct bude použit ke stažení některých startovacích beatmap pro vás.
+5. osu! vás vyzve k přihlášení nebo registraci.
+   - Pokud již účet máte, přihlaste se.
+   - Pokud se potřebujete zaregistrovat, podívejte se na [Registrace](/wiki/Registrace).
+
```
Comment (TPGPL) [inline wiki/Installation/cs.md:23]:
```suggestion

```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by Ondra-D** [comment]
you already fixed it

**Review by TPGPL — APPROVED** [review]

**Pull request merged**